tests/cmd/set-version/upgrade_compatible_dependency.toml

```diff
@@ -4,7 +4,7 @@ status = "success"
 stdout = ""
 stderr = """
     Upgraded cargo-list-test-fixture-dependency from 0.4.3 to 0.4.5
-Updated dependency cargo-list-test-fixture from 0.4.3 to 0.4.5
+Updated dependency cargo-list-test-fixture-dependency in cargo-list-test-fixture from 0.4.3 to 0.4.5
```

Comment:
Not a fan of this message change as it puts the focus on information preceding it.

I'd just do this as "Updated dependency in cargo-list-test-fixture from 0.4.3 to 0.4.5"